### ISSUE TITLE
Annotated reactome data for a target contains spaces in the keys 

### ISSUE THREAD

**Issue by peatroot** [opentargets/platform#280]
The response for `private/target/<geneId>` contains a field `reactome` for Reactome pathways information. This has a couple of issues:
* duplicate keys
* keys with spaces

For example, see https://api.opentargets.io/v3/platform/private/target/ENSG00000157764, which returns:

<img width="344" alt="screenshot 2018-11-12 at 15 05 19" src="https://user-images.githubusercontent.com/7376272/48355781-b9187d80-e68c-11e8-8344-11329c641c0f.png">

**Comment by afaulconbridge** [comment]
What should it return instead? Underscores instead of spaces? I believe it is still possible to access these in the current version using quotes around the field names, so it is still technically valid JSON (even if not stylistically good!).

**Comment by peatroot** [comment]
It's possible, but a mess. Most fields use snake case since the API is written in python. The duplication is the worse problem.

**Comment by d0choa** [comment]
@mkarmona could you quickly check if this is still an issue now that you're looking at the API?

**Comment by mkarmona** [comment]
It is not good having spaces as it makes things a bit more complicated without being necessary. By example, it cannot be serialised to parquet file format so you are forced either to replace it or to change serialisation.
About duplication, initially it does not make sense but it is worth having a look at it.

**Comment by mkarmona** [comment]
@cmalangone , can we focus this just to remove the spaces in order to get the proper data?

**Comment by cmalangone** [comment]
The ETL has fixed for the target.
Moreover there is a generic class that can be used in similar cases.
The reactome info is added with the proper fields with _ against "a space.

I tested the code with parquet files (in this case the input should be already correct due the fact that parquet file does not accept space for the key ) and I tested the code against the JSON dump.

Branch: cm_280_replace_fields_with_space_with_dash_target
Es: 20.02_target_280_j AND 20.02_target_280_p
